Cargo.lock

```diff
@@ -3296,7 +3296,7 @@ dependencies = [
  "httpdate",
  "itoa",
  "pin-project-lite",
- "socket2 0.5.7",
```

Comment:
:( this sucks

Reply:
huh, 9`hyper`'s dependency doesn't seem to have changed](https://github.com/hyperium/hyper/blame/618a18580ae14776c05ab6789a82b5c43281fe8b/Cargo.toml#L42), i guess Cargo Just Did That?

Reply:
yeah, cargo's behavior with wider dependency versions is just not great. Some day I should do a deep dive to figure out what's going on.